_stbt/core.py

```diff
@@ -738,7 +738,7 @@ def _match_all(self, image, frame, match_parameters, region):
             # Remove this copy once we make numpy_from_sample *not* unmap the
             # sample when the contextmanager exits, but when no further
             # references are held to the numpy image.
-            npframe = numpy.copy(npframe)
+            npframe = npframe.copy()
```

Comment:
This comment has now been implemented, but we still haven't removed the copy -- we should modify the comment to state when it will be safe to remove the copy.